### DISCUSSION TITLE
Plugin category

### DISCUSSION THREAD

**Discussion by dw199** [apohl79/audiogridder#1222] [General]
Hi,

Is there a way to change the category of a plugin in AGserver? I find that many of my plugins are placed by default in inconsistent or counterintuitive categories, which can be frustrating. It would be helpful to be able to change the category for each plugin manually, so I can see all my plugins in one place when I'm looking for a reverb, etc.

There's another similar feature in Logic which takes this a step further, which is custom category menus - where any given plugin can be put under multiple dropdown categories. I find this to streamline my workflow in an intuitive way. For example, a plugin like Soundtoys Effect Rack, which has multiple functions, can be found under delay, reverb, saturation, etc, so I can see it as an option when I'm looking for a creative solution to a problem.

I haven't found a way to do either of these things in Audiogridder, but I might be missing it. Is this already possible - or something that you would consider implementing?

Best
Dylan

**Comment by apohl79** [comment]
Hey Dylan, it is not supported as of today. But there exists a feature request for this. Thx.
